screens/ClockIn.js

```diff
@@ -301,9 +300,9 @@ class ClockIn extends React.Component {
     }
 
     let timeCardStatus = timecard.timeCardStatus
-    let authClientUserName =
-      this.props.navigation.state.params !== undefined &&
-      this.props.navigation.state.params.authClientUserName
+    // let authClientUserName =
```

Comment:
Why are they commented out? Is it because they are not used?

Reply:
Yes, when navigation still uses 4.x, these are not used, so remove now.